test/e2e/memtierd.test-suite/memtierd.source.sh

```diff
@@ -27,8 +27,12 @@ memtierd-os-env() {
 }
 
 memtierd-start() {
-    vm-pipe-to-file "memtierd.yaml" <<< "${MEMTIERD_YAML}"
-    vm-command "nohup sh -c 'socat tcp4-listen:${MEMTIERD_PORT},fork,reuseaddr - | memtierd -config memtierd.yaml -debug' > ${MEMTIERD_OUTPUT} 2>&1 & sleep 2; cat ${MEMTIERD_OUTPUT}"
+    if [ -z "${MEMTIERD_YAML}" ]; then
+        vm-command "nohup sh -c 'socat tcp4-listen:${MEMTIERD_PORT},fork,reuseaddr - | memtierd -prompt -debug' > ${MEMTIERD_OUTPUT} 2>&1 & sleep 2; cat ${MEMTIERD_OUTPUT}"
```

Comment:
As the command that launches memtierd is rather complex, let's avoid duplicating it (which would result in two complicated commands to maintain and make errors to them).

How about setting `MEMTIERD_OPTS=-config memtierd.yaml -debug` or `MEMTIERD_OPTS=-prompt -debug`, depending on `MEMTIERD_YAML` being empty? Then we could have a single vm-command that would run `... | memtierd $MEMTIERD_OPTS > ...`.

Reply:
Done.